packages/xo-web/src/xo-app/xosan/new-xosan.js

```diff
@@ -195,10 +248,12 @@ export default class NewXosan extends Component {
     this.setState({ brickSize })
   }
 
-  _selectSr = async (event, sr) => {
-    const selectedSrs = { ...this.state.selectedSrs }
-    selectedSrs[sr.id] = event.target.checked
-    this.setState({ selectedSrs })
+  _selectSrs = selectedSrs => {
+    if (some(selectedSrs, sr => this._getDisableSrCheckbox(sr) === true)) {
+      return error(_('xosanSrOnSameHost'), _('xosanSrOnSameHostMessage'))
```

Comment:
I don't think you want a notification to pop up everytime you select a SR that's not compatible. I think an error message above or below the table whenever you're in a "bad state" would be better.